pygeoapi-skin-dashboard/templates/collections/items/item.html

```diff
@@ -10,11 +10,14 @@
       <span style="white-space: nowrap;">{{ val | urlize() }}</span>
     {% elif v is mapping %}
       {% for i,j in v.items() %}
-        <i>{{ i }}:</i> {{ render_item_value(j, 60) }}<br/>
+      <span style="white-space: nowrap;"><i>{{ i }}:</i> {{ render_item_value(j, 60) }}</span><br/>
       {% endfor %}
     {% elif v is iterable %}
       {% for i in v %}
         {{ render_item_value(i, 60) }}
+        {% if not loop.last %}
+            <hr/> {# Divider between items #}
+        {% endif %}
```

Comment:
This was coming from the work in the previous PR. Unclear if this should be undone